Assets/Scripts/Interaction/Hand.cs

```diff
@@ -136,15 +145,18 @@ void updateSigilAnchor()
         if (IS_PRIMARY) return;
         currentProjectile = other.gameObject;
         if (getSpeed() > speedThreshold) {
-            if (!IS_PRIMARY)
+            if (!GetComponent<Hand>().IS_PRIMARY)
```

Comment:
It was fine before, you changed it to a worse version. This IS the Hand script, you don't need to get it.